src/client/containers/ProfilePage/ProfilePage.styles.css

```diff
@@ -47,70 +59,59 @@ div.quiz-results-header {
   padding-top: 3%;
 }
 
-.card-profile-result-list-items {
+.profile-quiz-container .card-profile-result-list-items {
   padding-right: 0;
   background-color: transparent !important;
 }
 
-li.card-profile-result-list-item {
+.profile-quiz-container li.card-profile-result-list-item {
   font-size: 1rem;
 }
 
-h2.card-profile-result-title {
+.profile-quiz-container h2.card-profile-result-title {
   font-size: 1.5625rem;
   margin-bottom: 0;
   padding: 0 3em 0 0;
   line-height: 1.875rem;
 }
 
-ul.card-profile-result-list-items {
+.profile-quiz-container ul.card-profile-result-list-items {
   padding-top: 0;
   padding-right: 0;
   min-height: 0;
 }
 
-.profile-career-header {
-  padding: 5% 0 0 3%;
-  height: 6em;
+.profile-career-container .profile-career-header {
+  padding: 3% 0 2% 4%;
+  height: 5.8em;
 }
 
 .profile-career-container {
   margin-left: 4%;
 }
 
 .profile-career-container .profile-career-body {
-  margin-bottom: 40px;
+  padding: 4% 0 0 4%;
```

Comment:
Can you add some bottom padding? Something between 2% - 4%, so there is some room between the triangle and the bottom margin

Reply:
fx. `padding: 4% 0 2% 4%`

Reply:
Hi @valmdz ,actually the triangle should lay beside the end of the sentence. @Poorooshka is working on it.For that sake i left it like that.Can i do it after push that triangle PR? It need any changes with some padding, thank you. 😊

Reply:
Yeah, that's true. The PR was sent yesterday.